swift/apple/FirezoneKit/Sources/FirezoneKit/Views/GrantVPNView.swift

```diff
@@ -17,7 +17,13 @@ final class GrantVPNViewModel: ObservableObject {
 
   func grantPermissionButtonTapped() {
     Log.app.log("\(#function)")
-    store.createVPNProfile()
+    Task {
```

Comment:
I moved the `Task {}` blocks to the UI code since that code should really be responsible for managing non-blocking calls, not the utility classes and functions they're calling.